mindjunkies/dashboard/tests/test_admin.py

```
@@ -0,0 +1,118 @@
+        assert teacher_admin.list_display == ("user", "full_name", "email", "phone", "address", "verified")
+        
+    def test_actions(self, admin_site):
+        teacher_admin = TeacherVerificationAdmin(TeacherVerification, admin_site)
+        assert "approve_teacher" in teacher_admin.actions
+        assert "disapprove_teacher" in teacher_admin.actions
+    
+    def test_approve_teacher_action(self, admin_site, teacher_group):
+        # Create a test user with a verification request
+        user = baker.make(settings.AUTH_USER_MODEL, is_teacher=False)
+        verification = baker.make(TeacherVerification, user=user, verified=False)
+        
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)